test/routes/github/release-created.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { initServer } from '../../../src/server';
+import { getConfig } from '../../../src/config';
+import { StreamLabs } from '../../../src/services/StreamLabs';
+import { TwitchChat } from '../../../src/services/TwitchChat';
+import { ReleaseCreatedPayloadBuilder } from '../../builders/release-created-payload-builder';
+import { ReleaseCreatedPayload } from '../../../src/schemas/github/release-created-payload';
+import { WebhookResponse } from '../../../src/schemas/webhook-response';
+
+describe('/github', () => {
```

Comment:
Can we add the HTTP method here?

Reply:
Fixed in https://github.com/streamdevs/webhook/pull/148/commits/c611c517113adfd021ff51e8858c8a5382d0decf even thought GitHub is not updating this diff